ee/ucp/admin/install/cloudproviders/install-on-azure.md

```diff
@@ -69,7 +70,7 @@ objects are being deployed.
 
 ### Azure Configuration File
 
-For Docker UCP to integrate with Microsoft Azure,each UCP node in your cluster
+For Docker UCP to integrate with Microsoft Azure, each UCP node in your cluster that runs `kubelet` component
```

Comment:
My nit `each Linux UCP node in your cluster`. People may think a worker node is a swarm worker, therefore no kubelet component will be deployed :)

Reply:
will adopt proposed change and submit a new PR on this.

Reply:
@olly, how about this:
```
each Linux UCP node in your cluster that runs `kubelet` component needs an Azure configuration file, `azure.json`. Note, that Swarm only nodes either Linux or Windows do not run `kubelet` component and therefore do not require `azure.json` file.
```